_content/blog/2024-11-19-Testing_Oskari_and_reporting_bugs.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: 'Testing Oskari and reporting bugs'
+excerpt: 'How Oskari is tested and how to report bugs.'
+date: 2024-11-19 13:00:00 +0200
+categories: [testing]
+tags:
+  - blog
+---
+
+# Testing Oskari and reporting bugs
+
+An article written by Ina Högström (trainee, NLS FI).
+
+I came on board to the National Land Survey’s team in the later stages of Oskari’s development cycle in January 2024. One of the new features that the team worked on was changing the thematic map feature to be done with React instead of jQuery: the whole user interface related to thematic maps was re-written with the new library. First I familiarised myself with [Paikkatietoikkuna](https://kartta.paikkatietoikkuna.fi/) and its demo version.
+
+Most of my testing I did on the demo version of Paikkatietoikkuna, which is updated to the newest version when there is a fix or a new implementation. Mostly my testing was centred around using thematic maps and problems that arise when using them. 
+
+## Thematic maps
+
+Thematic maps allow the user to present statistics on a map. The statistics are fetched from different data sources, e.g. via an API or from data the user has made themselves. The user can present indicators that either already exist or the ones they create. The indicators can be specified for example by year or by regional division. Some indicators have other specific options. Indicators can also be presented in a table or a bar chart. See an example below.
+
+![A screenshot of a thematic map](/content/blog/resources/2024/thematic1.png)
```

Comment:
Would "./resources/2024/thematic1.png" work? I'm not sure "/content/blog" works on any env (Github nor the actual site).
```suggestion
![A screenshot of a thematic map](./resources/2024/thematic1.png)
```

Reply:
Let's try